### ISSUE TITLE
Magic Cube, incorrect Face Handling

### ISSUE THREAD

**Issue by rummyr** [veeceeoh/xiaomi-hubitat#20]
Version 0.2.1b I think
It looks like the face extraction is not quite right, changing a couple of lines seems to fix it.
```groovy
private Map getMotionResult(String value) {
	String motionType = value[2..3]
	String binaryValue = hexToBin(value[0..1])
	Integer sourceFace = Integer.parseInt(binaryValue[2..4],2)
	Integer targetFace = Integer.parseInt(binaryValue[5..7],2)
```

**Comment by veeceeoh** [comment]
This issue should now be resolved in the newly released v0.3b of the driver, available [here](https://raw.githubusercontent.com/veeceeoh/xiaomi-hubitat/master/devicedrivers/xiaomi-cube-controller-hubitat.src/xiaomi-cube-controller-hubitat.groovy).

**Comment by saadalhamidi** [comment]
> Version 0.2.1b I think It looks like the face extraction is not quite right, changing a couple of lines seems to fix it.
> 
> ```groovy
> private Map getMotionResult(String value) {
> 	String motionType = value[2..3]
> 	String binaryValue = hexToBin(value[0..1])
> 	Integer sourceFace = Integer.parseInt(binaryValue[2..4],2)
> 	Integer targetFace = Integer.parseInt(binaryValue[5..7],2)
> ```

Hi, Can you help me, Where can I put in hubitat hub ?
